resources/web/wwi/FloatingWindow.js

```diff
@@ -0,0 +1,155 @@
+export default class FloatingWindow {
+  constructor(parentNode, name, url) {
+    this.name = name;
+    this.url = url;
+
+    this.floatingWindow = document.createElement('div');
+    this.floatingWindow.className = 'floating-window';
+    this.floatingWindow.id = name;
+    this.floatingWindow.style.visibility = 'hidden';
+    parentNode.appendChild(this.floatingWindow);
+    
+    this.floatingWindowHeader = document.createElement('div');
+    this.floatingWindowHeader.className = 'floating-window-header';
+    this.floatingWindow.appendChild(this.floatingWindowHeader);
+
+    this.headerText = document.createElement('p');
+    this.headerText.className = 'floating-window-text';
+    this.headerText.innerHTML = name;
+    this.floatingWindowHeader.appendChild(this.headerText);
+
+    this.headerQuit = document.createElement('button');
+    this.headerQuit.className = 'floating-window-close';
+    this.headerQuit.id = 'close-' + name;
+    this.headerQuit.innerHTML = ('&times');
+    this.floatingWindowHeader.appendChild(this.headerQuit);
+
+    this.floatingWindowContent = document.createElement('div');
+    this.floatingWindowContent.className = 'floating-window-content';
+    this.floatingWindow.appendChild(this.floatingWindowContent);
+
+    this.frame = document.createElement('iframe');
+    this.frame.id = this.name + '-window';
+    this.floatingWindowContent.appendChild(this.frame);
+
+    this._dragElement(this.floatingWindow);
+    this._maxSize(name);
+  }
+
+  getId() {
+    return this.floatingWindow.id;
+  }
+
+  getSize() {
+    return [this.floatingWindow.offsetWidth, this.floatingWindow.offsetHeight];
+  }
+
+  setSize(w, h) {
+    this.floatingWindow.style.width = w.toString() + 'px';
+    this.floatingWindow.style.height = h.toString() + 'px';
+  }
+
+  getPosition() {
+    return [this.floatingWindow.offsetLeft, this.floatingWindow.offsetTop];
+  }
+
+  setPosition(xPos, yPos) {
+    this.floatingWindow.style.left = xPos.toString() + 'px';
+    this.floatingWindow.style.top = yPos.toString() + 'px';
+  }
+
+  changeVisibility() {
+    if (this.floatingWindow.style.visibility === 'hidden') 
+      this.floatingWindow.style.visibility = 'visible';
+    else
+      this.floatingWindow.style.visibility = 'hidden';
+  }
+
+  setVisibility(visibility) {
+    this.floatingWindow.style.visibility = visibility;
+  }
+
+  getVisibility() {
+    return this.floatingWindow.style.visibility;
+  }
+
+  _dragElement(fw) {
+    let pos1 = 0, pos2 = 0, pos3 = 0, pos4 = 0;
```

Comment:
```suggestion
    let pos1 = 0;
    let pos2 = 0;
    let pos3 = 0;
    let pos4 = 0;
```

The coding style allows to declare several variables on the same line only if they are not initialized (`let pos1, pos2, pos3, pos4;`